lms/static/js/learner_dashboard/views/program_details_sidebar_view.js

```diff
@@ -25,23 +26,32 @@ class ProgramDetailsSidebarView extends Backbone.View {
         this.courseModel = options.courseModel || {};
         this.certificateCollection = options.certificateCollection || [];
         this.programCertificate = this.getProgramCertificate();
-        this.programRecordUrl = options.programRecordUrl;
         this.industryPathways = options.industryPathways;
         this.creditPathways = options.creditPathways;
         this.programModel = options.model;
-        this.programTabViewEnabled =  options.programTabViewEnabled;
+        this.subscriptionModel = options.subscriptionModel;
+        this.programTabViewEnabled = options.programTabViewEnabled;
+        this.isSubscriptionEligible = options.isSubscriptionEligible;
+        this.urls = options.urls;
         this.render();
     }
 
     render() {
-        const data = $.extend({}, this.model.toJSON(), {
-            programCertificate: this.programCertificate ?
-                this.programCertificate.toJSON() : {},
-            programRecordUrl: this.programRecordUrl,
-            industryPathways: this.industryPathways,
-            creditPathways: this.creditPathways,
-            programTabViewEnabled: this.programTabViewEnabled
-        });
+        const data = $.extend(
+            {},
+            this.model.toJSON(),
+            this.subscriptionModel.toJSON(),
+            {
+                programCertificate: this.programCertificate ?
+                    this.programCertificate.toJSON() : {},
+                industryPathways: this.industryPathways,
+                creditPathways: this.creditPathways,
+                programTabViewEnabled: this.programTabViewEnabled,
+                isSubscriptionEligible: this.isSubscriptionEligible,
+                arrowUprightIcon,
```

Comment:
Why does an icon get passed as part of object data? Seemed a little strange to me?

Reply:
as @jsnwesson correctly mentioned, we need to pass the icon to the template so it can render it

Reply:
Thanks for the explanation!